src/Kanvas/AccessControlList/Repositories/RolesRepository.php

```diff
@@ -59,4 +65,60 @@ public static function getAppRoles(AppInterface $app): Collection
     {
         return Role::where('scope', RolesEnums::getScope($app))->get();
     }
+
+    public static function getMapAbilityInModules(string $roleName): array
+    {
+        $roles = Bouncer::role()->where('name', $roleName)->firstOrFail();
```

Comment:
@FredPeal dont you need to filter the role by app?

Reply:
No Bouncer Apply the scope into the package